packages/block-library/src/pattern/block.json

```diff
@@ -13,6 +13,10 @@
 	"attributes": {
 		"slug": {
 			"type": "string"
+		},
+		"unsynced": {
+			"type": "boolean",
```

Comment:
This is nice! However, I think maybe we can try making it an "enum" so that we don't have to introduce more attributes for different syncing behaviors in the future. Possible options right now are: `sync` and `unsync`, but maybe we would need more?

Reply:
yeh, we might as well give ourselves some flexibility with that so have changed to an enum